pages/api/studies/index.js

```diff
@@ -0,0 +1,31 @@
+import dbConnect from '@lib/db-connect'
+import { generateKey } from '@lib/utils'
+import Study from '@models/study'
+
+export default async function handler(req, res) {

```

Comment:
Personally I am not a fan of having the API route specify the action, since the request method (POST in this case) tells you the action (creating a resource). If you feel strongly about this we should leave it to a vote though. What's really important is that we decide on endpoint naming conventions and stick to them.

Reply:
I agree it's better to let the url act as a resource path and http verb represent the action. I just suggested this to avoid having a single endpoint do multiple things. If we want to keep the convention of url -> resource & http verb -> action then that is fine by me. We can just have '/api/studies' accept multiple request types (e.g., POST, DELETE, etc...). Probably won't need GET since we can use the nextjs dynamic routes feature you've already used.